sportsref/nfl/players.py

```diff
@@ -177,6 +177,15 @@ def college(self):
         return college
 
     @sportsref.decorators.memoize
+    def collegeid(self):
```

Comment:
I guess maybe the name isn't clear, but the other function returns the college ID too, I'm pretty sure